src/main.rs

```diff
@@ -353,7 +352,7 @@ fn main() {
         }
     };
 
-    if vm_config.user_defined && vm_config.valid() {
+    if cmd_arguments.is_present("vm-config") && vm_config.valid() {
```

Comment:
This commit should be squashed into the original commit, so that we never introduce this boolean at all.

Reply:
Yep, good catch. This commit is gone now.